assets/js/state/selectors/user.js

```diff
@@ -0,0 +1,2 @@
+export const selectUserEmail = (state) => state.user.email;
```

Comment:
I think a generic `getUser` returning all the fields of the user could be more beneficial.
In no time, I will need to get the abilities as well, so I don't need to add a new selector.
Just a nitpick